mlflow/server/js/src/components/artifact-view-components/ShowArtifactMapView.js

```diff
@@ -0,0 +1,116 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { getSrc } from './ShowArtifactPage';
+import './ShowArtifactMapView.css';
+import { getRequestHeaders } from '../../setupAjaxHeaders';
+// There is problem with the marker icon so we loaded the latest css directly from unpkg
+// import 'leaflet/dist/leaflet.css';
+import L from 'leaflet';
```

Comment:
Can we depend on an older version of Leaflet to avoid this problem? It seems a bit weird to do this.

Reply:
Actually I see that there are some newer versions on the Leaflet website now too. Do those fix the problem?